src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,79 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        List<Integer> supplyAmounts = new ArrayList<>();
+        List<Integer> buyAmounts = new ArrayList<>();
+
+        readAndParseData(fromFileName, supplyAmounts, buyAmounts);
+
+        int[] statistics = calculateStatistics(supplyAmounts, buyAmounts);
+
+        writeResults(toFileName, statistics);
+    }
```

Comment:
This method seems to be doing too many things. It's reading and parsing data, calculating statistics, and writing results. Consider breaking it down into smaller methods, each responsible for a single task.